lib/locales/en.ts

```diff
@@ -428,6 +438,7 @@ export const locale: Locale = {
     florentius_of_strasbourg_bishop: 'Saint Florentius, Bishop',
     florian_of_lorch_and_companions_martyrs: 'Saint Florian and Companions, Martyrs',
     florian_of_lorch_martyr: 'Saint Florian, Martyr',
+    floscel_martyr: 'Saint Floscel, Martyr',
```

Comment:
I kind of dislike the `Floscel` name form in English. In Latin, they call him `Floscellus` (see [_Études sur les origines de lévêché de Bayeux_ (p559)](https://www.persee.fr/doc/bec_0373-6237_1868_num_29_1_462086#bec_0373-6237_1868_num_29_1_T1_0559_0000): _Passionem beati Floscelli, pueri […]_).

I’ve found several mentions of the `Floscellus` name in English:

- [WikiData](https://www.wikidata.org/wiki/Q113076838);
- [Behind the Name](https://www.behindthename.com/name/floscellus/submitted);
- [Text Manuscripts](https://www.textmanuscripts.com/medieval/psalter-soissons-atelier-237429).

Therefore, I suggest to use `floscellus_of_normandy_martyr` with the value `Saint Floscellus, Martyr`.

```suggestion
    floscellus_of_normandy_martyr: 'Saint Floscellus, Martyr',
```